samples/cellular/nrf_cloud_multi_service/src/cloud_connection.c

```diff
@@ -482,7 +494,10 @@ static void cloud_event_handler(const struct nrf_cloud_evt *nrf_cloud_evt)
 	case NRF_CLOUD_EVT_RX_DATA_GENERAL:
 		LOG_DBG("NRF_CLOUD_EVT_RX_DATA_GENERAL");
 		LOG_DBG("%d bytes received from cloud", nrf_cloud_evt->data.len);
-
+		device_deleted = (bool)nrf_cloud_evt->status;
+		if (device_deleted) {
+			LOG_INF("Device was removed from your account.");
+		}
```

Comment:
Nice as well

My only suggestion: Maybe some comments expounding upon this? I guess we are checking for nonzero status, but nonzero status on `EVT_RX_DATA` is somewhat unintuitive to me as a means to determine if we were deleted

Maybe it would make sense to create a new event, `NRF_CLOUD_EVT_USER_DELETED`, and fire that event from the caller of `NRF_CLOUD_EVT_RX_DATA_GENERAL`? (which could presumably detect this based on how it is setting the status)

Reply:
I thought about it. I'll take another look.

Reply:
the device receive an actual message notifying it of a deletion, see `nrf_cloud_disconnection_request_decode`

Reply:
the device receives an actual message indicating that the cloud has disconnected it.
i think it only gets sent when a device has been deleted?

Reply:
`(bool)nrf_cloud_evt->status;`
i don't think anything is set in the status field for a `NRF_CLOUD_EVT_RX_DATA_GENERAL` event.

Reply:
I added it

Reply:
ah i see the new commit.
i don't remember the reason we didn't make a new event for the disconnect message when it was initially implemented.
its seems like `NRF_CLOUD_EVT_RX_DATA_DISCON` would be worth adding.
and the comments could detail that the `"appID" : "DEVICE", "messageType" : "DISCON"` message is (currently only?) sent when a device is deleted from the cloud.

Reply:
i think SLM is the only sample that actually parses the RX data and checks for the discon message.
see `handle_cloud_cmd` in slm_at_nrfcloud.c

Reply:
Done.